src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,182 @@
+package duke.task;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class TaskManager {
+    static final int INDEX_OF_KEYWORD_OF_TASK = 0;
+    static final String SEPARATOR = " | ";
+
+    private ArrayList<Task> tasks = new ArrayList<>();
+
+    public void readInLine(String[] words) {
+        if (words[0].equals("T")) {
+            tasks.add(new Todo(words[2]));
+            if (words[1].equals("true")) {
+                tasks.get(Task.getNumberOfTasks() - 1).setDone(); //else leave isDone as false
```

Comment:
By splitting this line into two lines like this:
```
int lastTaskIndex = Task.getNumberOfTasks() - 1;
tasks.get(lastTaskIndex).setDone()
```
helps to improve code readability